be/src/storage/rowset/segment_rewriter.cpp

```diff
@@ -21,7 +22,8 @@ SegmentRewriter::~SegmentRewriter() {}
 
 Status SegmentRewriter::rewrite(const std::string& src_path, const std::string& dest_path, const TabletSchema& tschema,
                                 std::vector<uint32_t>& column_ids,
-                                std::vector<std::unique_ptr<vectorized::Column>>& columns, size_t segment_id) {
+                                std::vector<std::unique_ptr<vectorized::Column>>& columns, size_t segment_id,
+                                FooterPointerPB& partial_rowset_footer) {
```

Comment:
const FooterPointerPB&

Reply:
> const FooterPointerPB&

done，change in https://github.com/StarRocks/starrocks/pull/2969/commits/e7cecd1738b54e1f7019ba5d322823d03cca8b61